opencti-platform/opencti-graphql/src/modules/attributes/internalObject-registrationAttributes.ts

```diff
@@ -261,6 +261,31 @@ const internalObjectsAttributes: { [k: string]: Array<AttributeDefinition> } = {
     { name: 'account_lock_after_date', label: 'Account expiration date', type: 'date', mandatoryType: 'no', editDefault: false, multiple: false, upsert: false, isFilterable: true },
     { name: 'administrated_organizations', label: 'Organizations', type: 'string', format: 'short', mandatoryType: 'no', editDefault: false, multiple: false, upsert: false, isFilterable: true },
     { name: 'unit_system', label: 'Unit system', type: 'string', format: 'short', mandatoryType: 'no', editDefault: false, multiple: false, upsert: false, isFilterable: true },
+    { name: 'user_confidence_level',
+      label: 'User Confidence Level',
+      type: 'object',
+      format: 'standard',
+      mandatoryType: 'no',
+      editDefault: false,
+      multiple: false,
+      upsert: false,
+      isFilterable: true,
+      mappings: [
+        { name: 'max_confidence', label: 'Max Confidence', type: 'numeric', precision: 'integer', editDefault: false, mandatoryType: 'internal', multiple: false, upsert: false, isFilterable: true },
+        { name: 'overrides',
+          label: 'Overrides',
+          type: 'object',
+          format: 'nested',
+          editDefault: false,
+          mandatoryType: 'internal',
+          multiple: true,
+          upsert: true,
+          isFilterable: true,
+          mappings: [
+            { name: 'entity_type', label: 'Entity Type', type: 'string', format: 'short', editDefault: false, mandatoryType: 'external', multiple: false, upsert: false, isFilterable: true },
+            { name: 'max_confidence', label: 'Max Confidence', type: 'numeric', precision: 'integer', editDefault: false, mandatoryType: 'external', multiple: false, upsert: false, isFilterable: true },
+          ] },
+      ] },
```

Comment:
I'd like a review on this definition please @richard-julien 🙏

Reply:
reviewed in pair. I'd like to make sure the update cannot be done on incomplete objects, like 

```json
{
  "id": "26a02498-18e4-4b51-af18-44e5d8d6d5d9",
  "input": {
  "key": "user_confidence_level",
    "value": {
      "max_confidence": 96
  }
}
```
This is currently accepted and removes the overrides, which throws a GQL non-nullable field error on query